src/NinjaTools.FluentMockServer/Utils/IMockServerLogger.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using JetBrains.Annotations;
+
+namespace NinjaTools.FluentMockServer.Utils
+{
+    public interface IMockServerLogger
+    {
+        void WriteLine([NotNull] string message, params object[] args);
+        void Error([NotNull] string message, params object[] args);
+    }
+
+    internal class MockServerTestLogger : IMockServerLogger
+    {
+        /// <summary>
+        /// Gets the default <see cref="MockServerTestLogger"/>.
+        /// </summary>
+        public static IMockServerLogger Instance => Factory.Value;
+
+        private MockServerTestLogger(){}
+        private static readonly Lazy<IMockServerLogger> Factory;
+        static MockServerTestLogger()
+        {
+            Factory = new Lazy<IMockServerLogger>(() => new MockServerTestLogger());
+        }
+
+        /// <inheritdoc />
+        public void WriteLine(string message, [CanBeNull] params object[] args)
+        {
+            Console.WriteLine(message, args);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4927173)